Admin/BlockAdmin.php

```diff
@@ -243,9 +241,16 @@ public function getPersistentParameters()
      */
     public function toString($object)
     {
-        return $object->getName();
-    }
+        if (!is_object($object)) {
+            return '';
```

Comment:
Shouldn't there be an exception here ? I mean if an object is not an object, well…